README.md

```diff
@@ -1,5 +1,15 @@
 Anita & Jeri - Team FedAx
 
+> ## heroku URLs
+
+FedAx API
+- http://fedax.herokuapp.com
+- shipping quote endpoint: GET http://fedax.herokuapp.com/quote
+- shipping finalization endpoint: POST http://fedax.herokuapp.com/ship
+
+CosMade (bEtsy) online retailer (version that works with FedAx API)
+- http://cosmade.herokuapp.com
+
```

Comment:
Looking at the Markdown files in lib I'm wondering if those were meant to be included as documentation. If so directing folks to those files in the README could satisfy what I was talking about in my previous comment. If not, I'm curious as to why you have those lib markdown files.